components/cards/CardRowContainer.tsx

```diff
@@ -12,6 +12,7 @@ const CardRowContainer = ({ children }: Props) => {
         display: 'grid',
         gridTemplateColumns: {
           xs: '1fr',
+          md: '1fr',
```

Comment:
fyi @wheninseattle this will affect the home page which I think adhere to the tablet design.